crates/turborepo-lib/src/task_graph/mod.rs

```diff
@@ -169,9 +169,25 @@ macro_rules! set_field {
     }};
 }
 
+const LOG_DIR: &str = ".turbo";
+
 impl TaskDefinition {
+    pub fn workspace_relative_log_file(&self, task_name: &str) -> AnchoredSystemPathBuf {
+        let log_dir = AnchoredSystemPathBuf::from_raw(LOG_DIR)
```

Comment:
We can avoid an allocation since `join_component` constructs a new `PathBuf` for us.
```suggestion
        let log_dir = AnchoredSystemPath::from_raw(LOG_DIR)
```